site/app/templates/polls/ViewPoll.twig

```diff
@@ -76,18 +76,23 @@
                                    onchange="disableNoResponse();"
                             />
                             </td>
-                            <td class="option-text">
+                            <td class="option-text" id = "markdown-{{index}}">
                             {% include "misc/Markdown.twig" with {
                                 "content" : option.getResponse(),
                                 "style": "display: inline"
                             } only %}
+                            {% if ((poll.getAllowsCustomResponses() and option.isCustom()) and (user_id == option.getAuthorId())) %}
+                                <button type="button" class="move-btn delete-btn" aria-label="Delete Custom Response" style="border:0; padding:0;" onclick="removeCustomResponse({{ poll.getId() }}, {{ option.getId() }}, '{{ base_url }}');">
```

Comment:
It might be better to put each attribute on a separate line for clarity

Reply:
[5dd2702](https://github.com/Submitty/Submitty/pull/10197/commits/5dd2702545f62569aa64641a1f3bf874cc37e3ee)